pypxe/http.py

```diff
@@ -55,7 +62,9 @@ def handleRequest(self, connection, addr):
         self.logger.debug('  <--BEGIN MESSAGE-->\n\t{request}\n\t<--END MESSAGE-->'.format(request = repr(request)))
         startline = request.split('\r\n')[0].split(' ')
         method = startline[0]
-        target = startline[1]
+        req_file = startline[1]    
+        target = os.path.abspath(self.netbootDirectory + os.sep + startline[1])
```

Comment:
This and the other similar line in tftp.py probably want to use `os.path.join`: (`os.path.join(self.netbootDirectory, startline[1])`)